api/util/fileValidation.js

```diff
@@ -1,21 +1,40 @@
-const FileType = require('file-type');
+const sharp = require('sharp');
+
+const MAX_WIDTH = 700;
+const MAX_HEIGHT = 1100;
```

Comment:
Any reason for choosing this particular pixel width and height? Just curious.

Will constraining the width and height this way change the aspect ratio of uploaded images?

Should we specify a max DPI, as well?

Reply:
I picked those sizes because they fit inside a standard 8.5 x 11 page. It maintains aspect ratio. I just added one more setting in case it's too wide and too long.

We should be ok with dpi according to the dev who created the plugin "TIFF is pretty much the only bitmap format that cares about pixel density, thanks mainly to its use in the print industry, hence the tiff operation exposing xres and yres. JPEG doesn't deal with density, but some metadata added to JPEG can include values that might be interpreted as such - please see #967 for a related future possible enhancement."